static/styles/dark_theme.css

```diff
@@ -1,5 +1,7 @@
 @import "flatpickr/dist/themes/dark.css";
 
+$idle_color: hsl(29, 84%, 51%);
```

Comment:
Let's not change the color scheme in this PR if we can avoid it; that's an independent design change we're considering, and changing multiple things at once adds complexity.